nano/core_test/ipc.cpp

```diff
@@ -189,3 +189,19 @@ TEST (ipc, permissions_default_user_order)
 	nano::ipc::access access;
 	ASSERT_TRUE (access.deserialize_toml (toml));
 }
+
+TEST (ipc, invalid_endpoint)
+{
+	nano::system system (1);
+	system.nodes[0]->config.ipc_config.transport_tcp.enabled = true;
+	system.nodes[0]->config.ipc_config.transport_tcp.port = 24077;
```

Comment:
You should not need to set a specific port. Just let it choose a port automatically.

Reply:
It seems all tests in that file do it that way, not sure why... so we can leave it for now.